src/index.ts

```diff
@@ -148,10 +148,6 @@ class Radar {
     return VerifyAPI.trackVerified(params);
   }
 
-  public static trackVerifiedToken(params: RadarTrackParams = {}) {
```

Comment:
If we don't want this to be a breaking change, could we potentially pass the `params` here to track width `{ fraud: true }`?

Could also include a deprecation notice:
```js
Logger.warn('DEPRECATED - trackVerifiedToken will be removed in a future release.');
```

Reply:
Nobody uses this in prod yet (except for our demo page), so I think we're fine to skip this